packages/mcp-server/scripts/dev/pty-broker.py

```
@@ -0,0 +1,60 @@
+parser = argparse.ArgumentParser()
+parser.add_argument("--pidfile", required=True)
+parser.add_argument("cmd", nargs=argparse.REMAINDER)
+args = parser.parse_args()
+
+cmd = args.cmd[1:] if args.cmd[:1] == ["--"] else args.cmd
+if not cmd:
+    sys.exit("pty-broker: no command given")
+
+pid, master_fd = pty.fork()
+if pid == 0:
+    os.execvp(cmd[0], cmd)
```

Ruff (0.15.20)

[error] Starting a process without a shell

(S606)